src/content/en/ilt/pwa/e-commerce-lab-4-sw-precache-and-sw-toolbox.md

```diff
@@ -1,258 +0,0 @@
-project_path: /web/_project.yaml
```

Comment:
since this was deleted, we need a redirect for it - send it back to the main page or something other than a 404.